prometheus-to-sd/config/pod_config_test.go

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright 2018 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"testing"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+func TestIsMetricLabel(t *testing.T) {
+	for _, tc := range []struct {
+		desc     string
+		config   *podConfigImpl
+		label    string
+		expected bool
```

Comment:
want instead of expected

The same for other tests.

Reply:
done